package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/Index.js",
   "scripts": {
     "release": "eslint src && babel src --out-dir dist",
-    "dev": "cd docs && yarn && open 'http://localhost:8080' && NODE_ENV=development webpack-dev-server --inline --hot",
+    "dev": "cd docs && yarn && open 'http://localhost:8080' && NODE_ENV=development webpack-dev-server --inline --hot --host 0.0.0.0",
```

Comment:
this just allows us to view the docs from a vm or device via ip instead of just localhost.